db/migrate/20230525081252_update_created_manually_flag_on_customers.rb

```diff
@@ -2,24 +2,18 @@
 
 class UpdateCreatedManuallyFlagOnCustomers < ActiveRecord::Migration[7.0]
   class Customer < ApplicationRecord
-    has_many :orders, class_name: "Spree::Order"
-    acts_as_taggable
+    has_many :orders
   end
 
-  module Spree
-    class Order < ApplicationRecord
-      belongs_to :customer
-      self.table_name = 'spree_orders'
-    end
+  class Order < ApplicationRecord
+    self.table_name = 'spree_orders'
   end
 
-  def change
-    # We want to set the created_manually flag to true for all customers that don't have any orders
-    Customer.where.not(id: customers_with_at_least_one_order)
-      .update_all(created_manually: true)
+  def up
+    Customer.where.missing(:orders).update_all(created_manually: true)
```

Comment:
TIL there is a `.missing` method!

Reply:
Rubocop actually told me about it. I wrote the equivalent manually before:
```
left_out_joins(:orders).where(orders: { id: nil })
```